td-agent/Rakefile

```diff
@@ -160,6 +160,14 @@ def render_template(dest, src, config, opts={})
   end
 end
 
+def tar_command
+  if windows?
+    ["ridk", "exec", File.join(RubyInstaller::Runtime.msys2_installation.msys_path, "usr", "bin", "tar.exe")]
```

Comment:
Probably `["ridk", "exec", "tar"]` is enough for it.

Reply:
Ah, yes. I'll fix it.

Reply:
https://github.com/fluent/fluent-package-builder/pull/364/commits/942834e52d84ae0cedad0b2150a7620f9614cc92 probably will reduce a redundant line.